### ISSUE TITLE
Oxygen Builder & Cwickly Integration with Jet Smart Filters

### ISSUE THREAD

**Issue by paraimpacto** [CrocoBlock/suggestions#6244]
If at all possible, it would be great to develop an integration with both Oxygen and Cwickly.

**Comment by smithy5021** [comment]
I agree would love to see some clean unbloated compatibility and integration with Oxygen builder
